Flaky web_components_test failure

See https://github.com/web-platform-tests/wpt.fyi/actions/runs/22004095523/job/63583638323?pr=4729 (alternatively, if that's expired, [1_web_components_test.txt](https://github.com/user-attachments/files/25309679/1_web_components_test.txt)).

The test run ended with: 

```
2026-02-13T21:54:10.6391174Z chrome 138               ✓ components/test/wpt-app.html » <wpt-app> » WPTApp.prototype.* » wpt-results » testRuns from child wpt-results propagates to wpt-app testRuns
2026-02-13T21:54:43.4748967Z chrome 138               CALL title()
2026-02-13T21:54:43.4759493Z chrome 138               GET /session/:sessionID/title
2026-02-13T21:54:44.3630478Z chrome 138               RESPONSE title()
2026-02-13T21:55:10.3845633Z chrome 138               Tests failed: Error thrown outside of test function: Failed to read a named property 'WCT' from 'Window': Blocked a frame with origin "http://localhost:8081" from accessing a cross-origin frame.
2026-02-13T21:55:10.3847361Z   <unknown> at /components/wct-browser-legacy/browser.js:611
2026-02-13T21:55:10.3848138Z chrome 138               CALL quit()
2026-02-13T21:55:10.3853422Z chrome 138               DELETE /session/:sessionID
2026-02-13T21:55:10.4407448Z chrome 138               RESPONSE quit()
2026-02-13T21:55:10.4422723Z chrome 138               BrowserRunner complete
2026-02-13T21:55:10.4424255Z Test run ended in failure: Error thrown outside of test function: Failed to read a named property 'WCT' from 'Window': Blocked a frame with origin "http://localhost:8081" from accessing a cross-origin frame.
2026-02-13T21:55:10.4425887Z   <unknown> at /components/wct-browser-legacy/browser.js:611
2026-02-13T21:55:10.4426332Z 
2026-02-13T21:55:10.4426500Z chrome 138 (552/0/0 error)            
2026-02-13T21:55:10.4426931Z hook: cleanup
2026-02-13T21:55:10.4438762Z 
2026-02-13T21:55:10.4440058Z Error: Error thrown outside of test function: Failed to read a named property 'WCT' from 'Window': Blocked a frame with origin "http://localhost:8081" from accessing a cross-origin frame.
2026-02-13T21:55:10.4441505Z   <unknown> at /components/wct-browser-legacy/browser.js:611
2026-02-13T21:55:10.4441936Z 
2026-02-13T21:55:10.4705613Z npm notice
2026-02-13T21:55:10.4706636Z npm notice New major version of npm available! 10.8.2 -> 11.10.0
2026-02-13T21:55:10.4707664Z npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.10.0
2026-02-13T21:55:10.4708543Z npm notice To update run: npm install -g npm@11.10.0
2026-02-13T21:55:10.4709016Z npm notice
2026-02-13T21:55:10.4796826Z + stop_xvfb
2026-02-13T21:55:10.4797144Z + '[' true == true ']'
2026-02-13T21:55:10.4797469Z + killall Xvfb
2026-02-13T21:55:10.4812424Z make: *** [Makefile:146: web_components_test] Error 1
```

This didn't happen on [the later retry](https://github.com/web-platform-tests/wpt.fyi/actions/runs/22004095523/job/63584951855).

--- a/util/wct.sh
+++ b/util/wct.sh
@@ -23,6 +23,7 @@ fi
 cd webapp
 # Patch wct-browser-legacy to avoid cross-origin error in ChildRunner.get
 # See https://github.com/web-platform-tests/wpt.fyi/issues/4754
+# This workaround can be phased out if/when wct-browser-legacy is updated or replaced.
 sed -i 's/return window.parent.WCT._ChildRunner.get(target, true);/try { return window.parent.WCT._ChildRunner.get(target, true); } catch (e) { return null; }/' node_modules/wct-browser-legacy/browser.js
 if [ "$UID" == "0" ]; then
   # Used in .github/actions/make-in-docker/Dockerfile

--- a/util/wct.sh
+++ b/util/wct.sh
@@ -21,6 +21,9 @@ if [ "$USE_FRAME_BUFFER" == "true" ]; then
 fi
 
 cd webapp
+# Patch wct-browser-legacy to avoid cross-origin error in ChildRunner.get
+# See https://github.com/web-platform-tests/wpt.fyi/issues/4754
+sed -i 's/return window.parent.WCT._ChildRunner.get(target, true);/try { return window.parent.WCT._ChildRunner.get(target, true); } catch (e) { return null; }/' node_modules/wct-browser-legacy/browser.js
 if [ "$UID" == "0" ]; then
   # Used in .github/actions/make-in-docker/Dockerfile
   sudo -u browser npm test

